torch/testing/_internal/common_utils.py

```diff
@@ -2860,7 +2860,44 @@ def assertLeaksNoCudaTensors(self, name=None):
     def enforceNonDefaultStream(self):
         return CudaNonDefaultStream()
 
-    def assertExpectedInline(self, actual, expect, skip=0):
+    def _remove_ansi_escape(self, input):
+        # 7-bit C1 ANSI sequences
+        ansi_escape = re.compile(r'''
```

Comment:
I dislike the fact that this regex uses sequences that cross chartype boundaries (like "@-Z") but in general is reasonable.

Reply:
alternatively I can add a flag ignore_ansi_escape, like the other two flags.